cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,24 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('User', () => {
-  before(() => {
+import UserPageObject from '../support/pages/user.pageObject';
+
+const userPage = new UserPageObject();
 
+describe('User', () => {
+  let user;
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+    cy.register();
   });
 
   it.skip('should be able to follow the another user', () => {
-
+    cy.createUser(user.email, user.username);
+    userPage.visit(`#/@${user.username}/`);
+    userPage.followBtn.click();
```

Comment:
Directly clicking on page elements without any assertion before it might lead to flaky tests. It's a best practice to always assert the existence and the correct state of an element before interacting with it.